taxcalc/tests/test_functions.py

```diff
@@ -9,9 +9,89 @@
 import os
 import sys
 import re
-from taxcalc import IncomeTaxIO  # pylint: disable=import-error
+from taxcalc import IncomeTaxIO, Records  # pylint: disable=import-error
 from io import StringIO
 import pandas as pd
+import ast
+
+CUR_PATH = os.path.abspath(os.path.dirname(__file__))
+FUNCTIONS_PY_PATH = os.path.join(CUR_PATH, '..', 'functions.py')
+PYTHON3 = sys.version_info >= (3, 0)
```

Comment:
suggest using `six.PY2` as we already depend on `six`:

https://pythonhosted.org/six/